### DISCUSSION TITLE
Waiting a long time to create arrowsfile ,But it's still not finished

### DISCUSSION THREAD

**Discussion by WangHaoPlan** [GreenleafLab/ArchR#1857] [Questions / Documentation]
hello Let me talk about my problem：
I created myself geneAnnotation and genomeAnnotation for Non-model species ，when I create arrowsfile ,it let me waiting a long time, But it's still not finished，tmp file also no renovate.
Can I use any method to build arrow files？I think Archr is a very good software !
thank you for answer my question.
[ArchR-createArrows-387cd4858b7-Date-2023-02-24_Time-11-42-21.log](https://github.com/GreenleafLab/ArchR/files/10830201/ArchR-createArrows-387cd4858b7-Date-2023-02-24_Time-11-42-21.log)

**Comment by rcorces** [comment]
There is no error in your log file so it doesnt look like the function failed. The log file says it was running for ~60 minutes. Was it running for longer than that? It looks like your genome has >10,000 chromosomes/scaffolds which is far beyond anything I've seen used for ArchR. If anything were causing a problem, it would be that.
